ng2-components/ng2-alfresco-core/src/services/AlfrescoAuthentication.service.ts

```diff
@@ -20,16 +20,14 @@ import { LogService } from './log.service';
 import { SettingsService } from './settings.service';
 import { StorageService } from './storage.service';
 import { AlfrescoApiService } from './alfresco-api.service';
-import { AuthService } from './auth.service';
 
 /** @deprecated AlfrescoAuthenticationService is deprecated. Use AuthService instead */
 @Injectable()
-export class AlfrescoAuthenticationService extends AuthService {
+export class AlfrescoAuthenticationService {
```

Comment:
The idea was that both old and new service still function, and old one is raising a working. If we wrap AuthService into the AlfrescoAuthenticationService (and emulate apis) then both services will work the same.